pkg/cmd/server/start/master_args.go

```diff
@@ -199,7 +199,11 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 			Latest: args.ImageFormatArgs.ImageTemplate.Latest,
 		},
 
-		ProjectRequestConfig: configapi.ProjectRequestConfig{},
+		ProjectConfig: configapi.ProjectConfig{
+			DefaultNodeSelector:    args.ProjectNodeSelector,
```

Comment:
I think I'm against exposing this on `openshift start`.  In keeping with our goal of `openshift start` being the simplest possible toy, this exposure looks unnecessary.  Shall we pull it?